Runtimes/Overlay/Cxx/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+
+if(NOT APPLE)
+  add_subdirectory(cxxshim)
+endif()
```

Comment:
This doesn't seem correct: CxxShim headers are used on all of the platforms, including Apple ones.